consensus/clique/snapshot_test.go

```diff
@@ -37,6 +37,9 @@ type testerVote struct {
 	auth   bool
 }
 
+// blockPeriod is the default minimum difference between two consecutive block's timestamps
+var blockPeriod = uint64(15)
```

Comment:
I think we can get rid of this constant and directly inline `15` into the test that uses it. There's a single invocation, might as well clean out this altogether.

Reply:
changed.